### DISCUSSION TITLE
Is there a way to easily change the localization of the viewer ?

### DISCUSSION THREAD

**Discussion by soranoc** [mozilla/pdf.js#17567] [Q&A]
My website is supporting multiple languages as well as the annotation editor embedded in the PDFJs viewer. I would like the editor's labels to match my current selected language but I can't seem to find a way to change the language (in the viewer's options for example). Have I missed something or is the feature not implemented yet ?

PS: I found the "L10n" option in the PDFViewer options, but I can't wrap my head around how to use it.

**Comment by carloshpds** [comment]
@soranoc Maybe it helps:
https://github.com/mozilla/pdf.js/issues/17572#issuecomment-1921931525

**Reply by soranoc** [reply]
Thanks for the reply, however I'm struggling to understand how you can access the PDFViewerApplicationOptions property ? In my case it seems to always be undefined, and I can't seem to find where can I import it from the pdfjs-dist package.
